library/src/main/java/org/apache/apex/malhar/lib/utils/serde/BufferSlice.java

```diff
@@ -34,7 +32,7 @@
 public class BufferSlice extends Slice
 {
   private static final long serialVersionUID = -471209532589983329L;
-  public static final BufferSlice EMPTY_SLICE = new BufferSlice(ArrayUtils.EMPTY_BYTE_ARRAY);
+  public static final BufferSlice EMPTY_SLICE = new BufferSlice(null, 0, 0);
```

Comment:
This specific change causes SpillableMapImplTest to fail with null pointer exception (null buffer). This null buffer is passed to an append method in Tfile.java (belongs to hadoop-commons)